packages/cascara/src/private/Section/Section.js

```diff
@@ -13,7 +14,7 @@ const Section = ({ children, footer, header }) => {
   return (
     <section className={styles._}>
       {header && <header className={styles.Header}>{header}</header>}
-      <div className={styles.Content}>{children}</div>
+      <Role className={styles.Content}>{children}</Role>
```

Comment:
@rociobaena this component is not the root. It can stay as `div`

The component in Section that needs to be Role is actually the one on line 15. 
```
<Role as={as} /> // default prop for as = 'section'